element-template-generator/core/src/main/java/io/camunda/connector/generator/api/CliCompatibleTemplateGenerator.java

```diff
@@ -32,6 +32,9 @@ public interface CliCompatibleTemplateGenerator<IN> extends ElementTemplateGener
   /** Provides a usage description for the generator. This description is used in the CLI help. */
   String getUsage();
 
+  /** Scan the source and returns with operations (both supported and not supported). */
+  List<Operation> operations(IN input);
+
```

Comment:
This interface is only intended for the CLI tool, and it is designed to be high-level and not contain anything specific to individual generator implementations. Have you considered creating a new interface for invoking the HTTP-specific template generators (such as Postman/OpenAPI) instead of extending this generic one?